packages/variants/src/variants.ts

```diff
@@ -85,7 +93,24 @@ export function getNeutralVariant(theme: IPartialTheme): ITheme {
     bodyStandoutBackground: p.neutralLight,
     bodyFrameBackground: !fullTheme.isInverted ? p.neutralLight : p.neutralLighter,
     bodyFrameDivider: !fullTheme.isInverted ? p.neutralLight : p.neutralQuaternary,
-    variantBorder: !fullTheme.isInverted ? p.neutralQuaternaryAlt : p.neutralLighterAlt
+    variantBorder: !fullTheme.isInverted ? p.neutralQuaternaryAlt : p.neutralLighterAlt,
+
+    buttonBackground: p.neutralLighter,
```

Comment:
keep the button background styles grouped together for readability

Reply:
@phkuo  Updated ordering. Moving the disabled button slots together split up the buttonBorder from the buttonBorderDisabled, is that ok?